test/integration/spec/bandwidthprofile/publisherhints.js

```diff
@@ -3,13 +3,14 @@
 'use strict';
 
 const assert = require('assert');
-const { video: createLocalVideoTrack } = require('../../../../es5/createlocaltrack');
+const { video: createLocalVideoTrack, audio: createLocalAudioTrack } = require('../../../../es5/createlocaltrack');
 const defaults = require('../../../lib/defaults');
 const { Logger } = require('../../../../es5');
 const connect = require('../../../../es5/connect');
 const { createRoom, completeRoom } = require('../../../lib/rest');
 const getToken = require('../../../lib/token');
 const { isFirefox } = require('../../../lib/guessbrowser');
+const SECOND = 1000;
```

Comment:
This is not necessary.

Reply:
I think its usage does make it little bit more readable.

Reply:
I think this is not needed because the variable names indicate that the values are in milliseconds.